mail_tracking/models/mail_message.py

```diff
@@ -161,7 +171,7 @@ def _prepare_dict_failed_message(self, message):
         return {
             'id': message.id,
             'date': message.date,
-            'author_id': message.author_id.name_get()[0],
+            'author_id': self.author_id and self.author_id.name_get()[0] or '',
```

Comment:
I don't understand this change.

This is an `@api.model`, so by definition `self` shouldn't have a record attached. Also, `name_get()[0]` would return a tuple usually, but returning a tuple or an empty string doesn't seem to have much sense.